osu.Game.Modes.Taiko/Objects/Drawables/Pieces/RingPiece.cs

```diff
@@ -0,0 +1,111 @@
+﻿using OpenTK;
+using OpenTK.Graphics;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.Textures;
+using osu.Game.Graphics;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace osu.Game.Modes.Taiko.Objects.Drawables.Pieces
+{

```

Comment:
For all the "piece" classes you define, it'd be good if you added xml doc explaining what piece exactly they are. Can potentially also do that at the Drawable that contains all the pieces where its children are constructed.